### ISSUE TITLE
[question] time support

### ISSUE THREAD

**Issue by criskkky** [connercsbn/SimpleAdmin#2]
hi, your plugin supports !ban [name] [time]?

**Comment by connercsbn** [comment]
No, and no plans on supporting it at the moment, but not necessarily opposed to it either.

**Comment by connercsbn** [comment]
https://github.com/connercsbn/SimpleAdmin/commit/f9d4415ade927749351666b006e2c14870c99cdf
Added support for timed bans, gonna test tomorrow and publish if everything looks good.
